### ISSUE TITLE
Add a list on the map portal to select the day or nigth map

### ISSUE THREAD

**Issue by ebocher** [Ifsttar/NoiseCapture#149]
It would be nice to switch from day to night time.

**Comment by nicolas-f** [comment]
The shown level is currently the LA50 which is a median level over the week.

The future noise level attached to each hexagon will be la50, laeq and lden (day, evening, night) . In order to display "night time", we have to compute/store also the ln level.

Then in the "layer" choice dialog we will have  **la50**, **laeq** and **lden**, **ln**

here:

![image](https://cloud.githubusercontent.com/assets/1382241/26109386/bc63a5b8-3a4f-11e7-80d7-d5099fa54fa1.png)

Do you agree @Picaut ?

**Comment by Picaut** [comment]
Yes, I'm agree. This can be done easily, since Laeq and Lden are computed from Laeq1s.

**Comment by ebocher** [comment]
I think the name of the layers must be Day sound level, Night sound level ...
